types/guacamole-client/lib/Tunnel.d.ts

```diff
@@ -70,7 +70,7 @@ export abstract class Tunnel {
      *
      * @param data The data to send to the tunnel when connecting.
      */
-    connect(data: string): void;
+    connect(data?: string): void;
```

Comment:
This data seems optional (both from runtime testing and reading the source code): https://github.com/apache/guacamole-client/blob/754e9649f1fa0ba225ee42b56ded64bc283d17df/guacamole-common-js/src/main/webapp/modules/Tunnel.js#L34